target/linux/ath79/mikrotik/profiles/00-default.mk

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Profile/Default
+	NAME:=Default Profile (all drivers)
+	PACKAGES:= \
+		kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport nand-utils
```

Comment:
I wonder whether we need these packages; particularly ledtrig-usbport startles me.
However, i actually newer build with Default Profile ...

Reply:
I guess this profile is a reminiscence from the old times; when I've used it it has always lacked one packet or another (ath10k... whatever). I'm removing the ledtrig-usbport, it's not needed at all anywhere in the subtarget.